packages/sdks/snippets/gen1-remix/app/routes/custom-child.tsx

```diff
@@ -0,0 +1,42 @@
+import { Builder, builder, BuilderComponent } from '@builder.io/react';
+
+import { LoaderFunctionArgs } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+import { CustomHero } from './components/CustomHero';
+
+builder.init('ee9f13b4981e489a9a1209887695ef2b');
+
+Builder.registerComponent(CustomHero, {
+  name: 'CustomHero',
+  inputs: [],
```

Comment:
We don't need to provide an empty array, we can drop this altogether. If you can confirm that, go ahead and remove all empty `inputs: []`

Reply:
I tried this, but it is not rendering correctly. It is expecting the inputs.